packages/ketcher-react/src/script/ui/views/modal/Modal.container.ts

```diff
@@ -14,34 +14,41 @@
  * limitations under the License.
  ***************************************************************************/
 
-import React from 'react'
 import { omit } from 'lodash/fp'
 import { connect } from 'react-redux'
-import modals from '../../dialog'
+import { Dispatch } from 'redux'
+import { DialogCallProps } from '../components'
```

Comment:
Modal container shouldn't know about any other props. It is out of his responsibility.

Reply:
I'm a bit confused here because I used TopToolbar.container.ts as an example and it knows about other props...
![image](https://user-images.githubusercontent.com/45556539/108967791-fdc75780-7690-11eb-8e5b-b38ed00e13f7.png)

Reply:
TopToolbar.container knows abot TopToolbar props, and Modal.container knows about Modal props but not about Dialog...

Reply:
Okay, will fix